dm/worker/config.go

```diff
@@ -171,6 +170,11 @@ func (c *Config) adjust() error {
 		}
 	}
 
+	if c.Name == "" {
+		fmt.Printf("worker name is not given, we will set AdvertiseAddr %s as the worker name\n", c.AdvertiseAddr)
```

Comment:
if can't use log instead. I think we can remove this line

Reply:
What's the reason for removing this?

Reply:
it will print to stdout, just seems stranger, you can keep it